project/threads/serializers.py

```diff
@@ -0,0 +1,339 @@
+from accounts.models import Profile
+from core.constants import CIVI_TYPES
+from rest_framework import serializers
+
+from .api import UpdateProfileImage
+from .models import Activity, Category, Civi, CiviImage, Thread
+
+WRITE_ONLY = {"write_only": True}
+
+class CategoryListSerializer(serializers.ModelSerializer):
+    """ """
+    class Meta:
+        model = Category
+        fields = ("id", "name")
+
+

```

Comment:
Category-related code should not be moved as part of this pull request. We have a separate task to migrate category-related code #1040 
```suggestion
```